gkctl/main.c

```diff
@@ -322,5 +322,5 @@ main(int argc, char *argv[])
 close_sock:
 	close(sock_fd);
 out:
-	return ret;
+	return 0;
```

Comment:
You want to add the line `ret = 0;` after the line `printf("%s\n", recv_buff);`. Otherwise, `gkctl` will always return zero.